### ISSUE TITLE
`cs-twisted-products-test` failure

### ISSUE THREAD

**Issue by gheber** [gheber/kenzo#138]
```
Failure Details:
 --------------------------------
 SMGR-FIBRATION []: 
      Unexpected Error: #<UNDEFINED-FUNCTION SMGR-FIBRATION {1004899183}>
The function CAT-9:SMGR-FIBRATION is undefined...
 --------------------------------
```

**Comment by sphyynx** [comment]
Francis Sergeraert => Gerd Heber.
==================================================

/> Failure Details:
 >  --------------------------------
 >  SMGR-FIBRATION []:
 >       Unexpected Error: #<UNDEFINED-FUNCTION SMGR-FIBRATION {1004899183}>
 > The function CAT-9:SMGR-FIBRATION is undefined...
 >  --------------------------------
/
Same answer: no identifier SMGR-FIBRATION in my Kenzo-9.

   Seems you  have let  Kenzo-8 files mixed  with Kenzo-9?  For example
SMGR-FIBRATION    was   a    function   in    CL-SPACE-EFHM.lisp   and
CS-TWISTED-PRODUCTS.lisp of Kenzo-8, files not in Kenzo-9.

Bien cordialement.

**Comment by gheber** [comment]
Fixed in e1058d2
